### PR TITLE
Add part side information as planned

### PR DESCRIPTION
**Pull request by bs-jokri**
Adds model already released in deprecated repository

### CODE REVIEW

**Comment by bs-jokri** [comment]
@jacewski-bosch could you please check

File: io.catenax.part_site_information_as_planned/1.0.0/PartSiteInformationAsPlanned.ttl [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+@prefix bamm: <urn:bamm:io.openmanufacturing:meta-model:1.0.0#>.
+@prefix unit: <urn:bamm:io.openmanufacturing:unit:1.0.0#>.
+@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:1.0.0#>.
+@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:1.0.0#>.
+@prefix : <urn:bamm:io.catenax.part_site_information_as_planned:1.0.0#>.
+
+:UUIDv4RegularExpression a bamm-c:RegularExpressionConstraint;
+    bamm:name "UUIDv4RegularExpression";
+    bamm:preferredName "Catena-X Id Regular Expression"@en;
+    bamm:description "The provided regular expression ensures that the UUID is composed of five groups of characters separated by hyphens, in the form 8-4-4-4-12 for a total of 36 characters (32 hexadecimal characters and 4 hyphens)."@en;
```
Comment (jacewski-bosch) [inline io.catenax.part_site_information_as_planned/1.0.0/PartSiteInformationAsPlanned.ttl:36]:
```suggestion
    bamm:description "The provided regular expression ensures that the UUID is composed of five groups of characters separated by hyphens, in the form 8-4-4-4-12 for a total of 36 characters (32 hexadecimal characters and 4 hyphens), optionally prefixed by \"urn:uuid:\" to make it an IRI."@en;
```

File: io.catenax.part_site_information_as_planned/1.0.0/PartSiteInformationAsPlanned.ttl [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    bamm:value "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$".
```
Comment (jacewski-bosch) [inline io.catenax.part_site_information_as_planned/1.0.0/PartSiteInformationAsPlanned.ttl:37]:
```suggestion
    bamm:value "(^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$)|(^urn:uuid:[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$)";
    bamm:see <https://datatracker.ietf.org/doc/html/rfc4122>.
```

**Comment by bs-jokri** [comment]
@jacewski-bosch accepted your suggestions. ok with merging?

**Comment by jacewski-bosch** [comment]
@bs-jokri yes thank you, please merge.

**Review by jacewski-bosch — APPROVED** [review]

**Pull request merged**